pom.xml

```diff
@@ -164,6 +164,11 @@
         <artifactId>proto-google-common-protos</artifactId>
         <version>${google.common-protos.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core</artifactId>
+        <version>${google.core.version}</version>
```

Comment:
`<scope>test</scope>`

Reply:
`<scope>test</scope>`  is already there,you can see here https://github.com/athakor/java-container/pull/1/files#diff-8797f982f4fa20aed2356c731473ea6eR74